systems/console-bff/common/data/correspondent.ts

```diff
@@ -0,0 +1,227 @@
+import { TCorrespondentsMap } from "../types";
```

Comment:
This is related to MobileMoney are we using it here?

Reply:
Addressed

Reply:
addressed.